- Disable valgrind tests due to hangs, x86-only and missing TAppDecoder
- Disable encode tests to avoid circular dependency on ffmpeg

--- tests/Makefile.am.orig	2020-04-22 15:53:12 UTC
+++ tests/Makefile.am
@@ -1,18 +1,7 @@
 
 TESTS = $(check_PROGRAMS) \
     test_external_symbols.sh \
-    test_gop.sh \
-    test_interlace.sh \
-    test_intra.sh \
-    test_invalid_input.sh \
-    test_mv_constraint.sh \
-    test_owf_wpp_tiles.sh \
-    test_rate_control.sh \
-    test_slices.sh \
-    test_smp.sh \
-    test_tools.sh \
-    test_weird_shapes.sh \
-	test_pu_depth_constraints.sh 
+    $(NULL)
 
 EXTRA_DIST = \
     test_external_symbols.sh \
